src/GitLinkTask/GitLink.targets

```diff
@@ -10,6 +10,7 @@
         PdbFile="$(IntermediateOutputPath)$(TargetName).pdb"
         Method="$(GitLinkMethod)"
         SkipVerify="$(GitLinkSkipVerify)"
+        IndexAllDepotFiles="$(GitLinkIndexAllDepotFiles)"              
```

Comment:
Can we set this property to a default based on `Condition=" '$(MSBuildProjectExtension)' == '.vcxproj' "` at least, so that native PDBs "just work"?